### DISCUSSION TITLE
Convert gfm to pdf?

### DISCUSSION THREAD

**Discussion by wyttenbach** [quarto-dev/quarto-cli#11484] [Q&A]
### Description

Is quarto a good tool to convert gfm to pdf?
It doesn't seem to understand the gfm extension:
```
$ quarto render process.gfm --to pdf
ERROR: Can't determine execution engine for process.gfm
...
```

**Comment by mcanouil** [comment]
Quarto inputs are: markdown (md/qmd) and Jupyter notebook (ipynb).

Note that special GitHub syntax won't work even if you change the file extension.

**Comment by cderv** [comment]
> Is quarto a good tool to convert gfm to pdf?

No Quarto is not a conversion tool. You need to use Pandoc directly for this. `pandoc --from gfm --to pdf process.gfm`

Quarto takes `.md` file or `.qmd` file using the Pandoc's Markdown syntax + extended syntax from quarto itself to render to various format. 

It may not understannd specific GFM feature that do not overlap with default Markdown syntax.
